src/Colourista/Pure.hs

```diff
@@ -142,6 +147,10 @@ cyan = withColourMode $ fromString $ setSGRCode [SetColor Foreground Vivid Cyan]
 {-# SPECIALIZE cyan :: HasColourMode => Text       #-}
 {-# SPECIALIZE cyan :: HasColourMode => ByteString #-}
 
+-- | Code to apply any arbitrary hex color for the terminal output.
```

Comment:
Could you add some documentation examples of this function usage with hexadecimal literals? So people will be aware of the recommended ways to use this function 🙂